types/jsonwebtoken/v7/package.json

```diff
@@ -0,0 +1,64 @@
+{
+    "private": true,
+    "name": "@types/jsonwebtoken",
+    "version": "7.4.3",
```

Comment:
📝 After release from [7.4.3 to 8.0.0](https://github.com/auth0/node-jsonwebtoken/compare/v7.4.3...v8.0.0), SignOptions.expiresIn throw an error when given undefined.